### PR TITLE
Add services offers

### PR DESCRIPTION
**Pull request by frwickst**

### CODE REVIEW

File: services/management/commands/turku_service_import/units.py [hunk 1]
```diff
@@ -177,6 +185,38 @@ def _handle_services_and_service_nodes(self, obj, unit_data):
...
+            offer_name = service_offer.get('nimi_kieliversiot', {})
+            for language, value in offer_name.items():
+                # Make sure that we have a string as the default value
+                if not descriptions[language]:
+                    descriptions[language] = ''
+
+                if not touched[language]:
+                    # Add spacing and header for service offers
+                    descriptions[language] = clean_text(descriptions[language])
+                    descriptions[language] += '\n\n' + SERVICE_OFFERS_TRANSLATIONS[language] + ':\n'
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/units.py]: `clean_text()` can return `None` (which is bad IMO), in which case this row will error.

**Comment by frwickst** [comment]
@tuomas777 Fixed this and made the changes that we talked about regarding the description and `palvelut`

File: services/management/commands/turku_service_import/units.py [hunk 2]
```diff
@@ -23,6 +24,12 @@
     '6': {'kuvaus_kieliversiot': 'www'},
 }
 
+SERVICE_OFFERS_TRANSLATIONS = {
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/units.py]: Still saying "offers" here.

File: services/management/commands/turku_service_import/units.py [hunk 3]
```diff
@@ -177,6 +185,43 @@ def _handle_services_and_service_nodes(self, obj, unit_data):
...
+        touched = {
+            'fi': False,
+            'sv': False,
+            'en': False,
+        }
+
+        for service_offer in unit_data.get('palvelutarjoukset', []):
+            for service_data in service_offer.get('palvelut', []):
+
+                offer_name = service_data.get('nimi_kieliversiot', {})
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/units.py]: This should also be `service_name` to not confuse anyone any more than is necessary in this thing :)

**Review by tuomas777 — COMMENTED** [review]
Also one commit message is talking about adding palvelutarjoukset names. Other than those naming issues, looks very nice! 👍

**Comment by frwickst** [comment]
@tuomas777 fixed

**Review by tuomas777 — APPROVED** [review]

**Pull request merged**